Assets/Scripts/Environment/InteractableObject.cs

```diff
@@ -5,12 +5,7 @@ public class InteractableObject : MonoBehaviour
     // Set this OnEnter and OnExit to avoiding repeatedly comparing tags
     public bool IsPlayerColliding { get; protected set; }
 
-    [SerializeField] new protected Collider collider;
-
-    protected void SetPlayerIsColliding(Collider other)
-    {
-        IsPlayerColliding = other.CompareTag(PlayerConstants.PlayerTag);
-    }
+    [SerializeField] new public Collider Collider;

```

Comment:
Because this is public, it's redundant now. But doesn't bother me. Remove it if you want, or don't, I don't mind. 

![image](https://user-images.githubusercontent.com/61809127/120889921-1da2db80-c5ef-11eb-8a0d-4141f33aa987.png)